ci-operator/config/openstack-k8s-operators/dataplane-operator/openstack-k8s-operators-dataplane-operator-main.yaml

```diff
@@ -77,9 +77,11 @@ tests:
     owner: openstack-k8s-operators
     product: ocp
     timeout: 1h0m0s
-    version: "4.10"
+    version: "4.12"
```

Comment:
So this version mentioned in this file references the install-config.yaml here:
https://github.com/openshift/release/blob/master/clusters/hive/pools/openstack-k8s-operators/install-config-hybrid-3cp_secret.yaml

Is my understanding correct?

Reply:
@bshephar yes, this is the secret `install-config-hybrid-3cp` that we reference in this cluster pool: https://github.com/openshift/release/blob/master/clusters/hive/pools/openstack-k8s-operators/openstack-k8s-operators-4-12-vexxhost-hybrid_clusterpool.yaml#L22
which has the label `version: "4.12"` used by the CI to select the correct pool.